<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="newGameButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>新游戏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>80</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
    <widget class="QPushButton" name="blackButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>黑(先手)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="whiteButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>130</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>白(后手)</string>
     </property>
    </widget>
    <widget class="QLabel" name="askingColorLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请选择您的棋子颜色</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="recordButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>180</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>存档</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>130</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>读档</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>350</y>
      <width>191</width>
      <height>181</height>
     </rect>
    </property>
    <widget class="QPushButton" name="blackButton2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>黑</string>
     </property>
    </widget>
    <widget class="QPushButton" name="whiteButton2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>白</string>
     </property>
    </widget>
    <widget class="QLabel" name="askingColorLabel2">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>60</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请选择您的棋子颜色</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="showingLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>当前黑方执子</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="retryButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>230</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>重新落子</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>280</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>对方落子</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>181</width>
      <height>271</height>
     </rect>
    </property>
    <widget class="QPushButton" name="Level0Button">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>简单</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Level1Button">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>中等</string>
     </property>
    </widget>
    <widget class="QLabel" name="askingLevelLabel">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>10</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请选择游戏难度</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="Level2Button">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>困难</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="replaceButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>280</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>AI辅助落子</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
